### ISSUE TITLE
ui5-link: remove unnecessary disabled attribute

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2673]
**Describe the bug**
Currently, when we have a disabled link, both aria-disabled and disabled attributes are rendered - one of them should be removed. It is always best to leave the 'native' attribute, so try to remove the aria-disabled attribute and test if the disabled information is still presented to the screen reader software user.

**To reproduce**
Steps to reproduce the behaviour:
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Link/
2. Inspect the disabled link
